subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/GradleBuildExperimentRunner.java

```diff
@@ -190,7 +222,9 @@ private GradleScenarioDefinition createScenarioDefinition(GradleBuildExperimentS
             experimentSpec.getDisplayName(),
             (GradleBuildInvoker) invocationSettings.getInvoker(),
             new GradleBuildConfiguration(gradleDistribution.getVersion(), gradleDistribution.getGradleHomeDir(), Jvm.current().getJavaHome(), actualJvmArgs, false, invocationSpec.getClientJvmArguments()),
-            experimentSpec.getInvocation().getBuildAction(),
+            buildSpec.isUseAndroidStudio()
```

Comment:
This can be removed once https://github.com/gradle/gradle-profiler/pull/394 is merged and released.